app/graph/types/comment_type.rb

```diff
@@ -3,6 +3,17 @@
   description 'A comment a user left on some content.'
   global_id_field :id
 
-  field :author, !UserType, property: :user
-  field :content, !ContentType
+  field :body, !types.String
+  field :author, !UserType
+
+  field :content do
+    type ContentType
+    resolve -> (comment, _, _) {
+      if comment.content_type == 'Photo'
+        Photo.find(comment.content_id);
```

Comment:
This should also not be nullable.
This will make the same query as doing `comment.content` so I think the entire field could just be the same as it was.

```ruby
field :content, !ContentType
```

Reply:
Hmm, it seems this is what you changed. Why wasn't it working beforehand? I was 🎩ing and it seems fine the way it was.

Reply:
> Does the ContentType need a resolve to resolve which of the possible types to use?

This function on the schema should do that for us. https://github.com/GraphQL-Query-Planner/webscale/blob/master/app/graph/app_schema.rb#L5

Reply:
Should https://github.com/GraphQL-Query-Planner/webscale/blob/master/app/graph/app_schema.rb#L5 just resolve the proper type automatically? Still getting errors with the following query. 

```
query {
  comment(id: "gid://webscale/Comment/1") {
    body,
    content {
      photo_url
    }
  }
}
```

Reply:
What's the error?

Reply:
Removed the `,` same issue: 

See the [screenshot](https://github.com/GraphQL-Query-Planner/webscale/pull/23#issuecomment-339699342) above

Reply:
Doubt this helps, but: 

<img width="961" alt="screen shot 2017-10-30 at 8 24 04 pm" src="https://user-images.githubusercontent.com/6130700/32202026-555a2b64-bdb0-11e7-9b40-8f7f5d49a3f2.png">